pkg/remoteconfig/state/repository.go

```diff
@@ -256,8 +262,8 @@ func (r *Repository) Update(update Update) ([]string, error) {
 	}
 
 	changedProducts := make([]string, 0)
-	for product, configs := range result.changed {
-		if len(configs) > 0 {
+	for product, updated := range result.productsUpdated {
+		if updated {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** Is that necessary? If a key is in the map, it means the product has been updated

Reply:
It's to be future-proof, and it is not really costly